docs/docsite/rst/aws_ec2_guide.rst

```diff
@@ -0,0 +1,523 @@
+.. _ansible_collections.amazon.aws.docsite.dynamic_inventory:
+
+
+Dynamic Inventory Plugin
+========================
+
+``aws_ec2`` dynamic inventory plugin allow users to point at data sources to compile the inventory of AWS EC2 hosts that Ansible uses to target tasks, either via the ``-i /path/to/file`` and/or ``-i 'host1, host2'`` command line parameters or from other configuration sources.
```

Comment:
Something feels off about this, it's halfway between explaining what a dynamic inventory does, and what the AWS EC2 plugin does, but doesn't quite do either.

I'd probably lean towards this first paragraph explaining what a dynamic inventory plugin *generally* does, and slightly tweaking the paragraph below below to explain specifically what the AWS EC2 plugin does: talk to the AWS APIs to pull a list of EC2 instances.

Reply:
@tremble Tried to improve the doc. Let me know if there's something else to improve.